sgqlc/types/__init__.py

```diff
@@ -2016,6 +2018,25 @@ def type(self):
         self._type = self._type.resolve(self.schema)
         return self._type
 
+    @staticmethod
+    def _to_python_name(graphql_name):
+        '''Converts a GraphQL name, ``aName`` to Python: ``a_name``.
+
+        Note that an underscore is appended if the name is a Python keyword.
+        >>> BaseItem. _to_python_name('aName')
+        a_name
+        >>> BaseItem. _to_python_name('for')
+        for_
```

Comment:
It was failing because of some docstring nuances:

1. you need an empty line between the doc text and the test (`>>>` is assumed to be a continuation of the `... is a Python keyword`)
2. my mistake: since the return of `_to_python_name()` is a string, we need quotes around it 
3. You left a space between `BaseItem.` and `_to_python_name` (not a blocker, does not fail the test)

```suggestion
        Note that an underscore is appended if the name is a Python keyword.

        >>> BaseItem._to_python_name('aName')
        'a_name'
        >>> BaseItem._to_python_name('for')
        'for_'
```

Reply:
I can edit you commit and merge, but if you wish to push-force your branch with that, I'll merge afterwards.

Reply:
Ah, makes sense--thanks!